.editorconfig

```diff
@@ -0,0 +1,34 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+tab_width = 4
+trim_trailing_whitespace = true
+
+[*.gradle]
+indent_style = tab
+
+[*.java]
+indent_style = tab
+
+[*.json]
+indent_style = space
+indent_size = 2
+
+[quilt.mod.json]
```

Comment:
Why the double `json` based declarations?

Reply:
QMJs are indented with tabs, while everything else (which would be Minecraft's data) is indented with 2 spaces